modules/plot/main/coffee/Expose.coffee

```diff
@@ -7,6 +7,14 @@ hx.plot.arcCurve = arcCurve
 hx.plot.svgCurve = svgCurve
 hx.Axis = Axis
 hx.Graph = Graph
+
+hx.graph = (options) ->
+  selection = new hx.detached 'div'
+  graph = new Graph selection.node(), options
+  # There is no point rendering it now, the selection is of zero size.
+  selection.on 'resize', -> graph.render()
```

Comment:
We need to do this as the selection will be initially of zero height/width

We might want to think about doing this for all graphs ? Would it be expensive ?